src/App.tsx

```diff
@@ -1,20 +1,31 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
+import { Navigate, Route, Routes } from 'react-router-dom';
 
 import './App.scss';
+import { NavBar } from './components/NavBar';
+import { HomePage } from './components/HomePage';
+import { NotFoundPage } from './components/NotFoundPage';
+import { PeoplePage } from './components/PeoplePage';
 
 export const App = () => {
   return (
     <div data-cy="app">
-      <Navbar />
+      <NavBar />
 
-      <div className="section">
+      <main className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="*" element={<NotFoundPage />} />
```

Comment:
The `NotFoundPage` is set as the default route. It should only be displayed if no other route matches. Reorder the routes to fix this.